src/wp-admin/includes/upgrade.php

```diff
@@ -2349,6 +2353,36 @@ function upgrade_640() {
 	}
 }
 
+/**
+ * Executes changes made in WordPress 6.5.0.
+ *
+ * @ignore
+ * @since 6.5.0
+ *
+ * @global int  $wp_current_db_version The old (current) database version.
+ * @global wpdb $wpdb                  WordPress database abstraction object.
+ */
+function upgrade_650() {
+	global $wp_current_db_version, $wpdb;
+
+	if ( $wp_current_db_version < 57200 ) {
+		$stylesheet = get_stylesheet();
+
+		// Set autoload=no for the previous all themes.
```

Comment:
Confusing/outdated comment

```suggestion
		// Set autoload=no for all themes except the current one
```